.github/workflows/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Issue/Suggestion: Target the MPL's GHW directory.
```suggestion
    directory: ".github/workflows"
```